samples/drivers/video/capture/prj.conf

```diff
@@ -5,3 +5,4 @@ CONFIG_PRINTK=y
 CONFIG_LOG=y
 CONFIG_DISPLAY=y
 CONFIG_REQUIRES_FLOAT_PRINTF=y
+CONFIG_LOG_MODE_DEFERRED=y
```

Comment:
IMHO it makes sense as a hotfix, but can be interesting to fix it device-side by increasing the stack size.

`CONFIG_LOG_MODE_DEFERRED` can drop lines of log, which can make console-based test harness less reliable.